lib/config.js

```diff
@@ -16,54 +16,47 @@ const babel = {
     'babel-preset-stage-0',
     'babel-preset-react',
   ].map(require.resolve),
-  plugins: [
-    'babel-plugin-styled-components'
-  ].map(require.resolve)
+  plugins: ['babel-plugin-styled-components'].map(require.resolve),
 }
 
 const rules = [
   {
-    test: /\.js$/,
+    test: /\.jsx?$/,
     exclude: /node_modules/,
     loader: require.resolve('babel-loader'),
-    options: babel
+    options: babel,
   },
   {
-    test: /\.js$/,
+    test: /\.jsx?$/,
```

Comment:
This package doesn't use the `.jsx` extension, so this change shouldn't be necessary